sequencer/src/api/sql.rs

```diff
@@ -619,45 +624,31 @@ async fn fee_header_dependencies<Mode: TransactionMode>(
     Ok(accounts)
 }
 
-async fn reward_header_dependencies<Mode: TransactionMode>(
-    catchup: &mut NullStateCatchup,
-    tx: &mut Transaction<Mode>,
+async fn reward_header_dependencies(
     instance: &NodeState,
     leaves: impl IntoIterator<Item = &Leaf2>,
 ) -> anyhow::Result<HashSet<RewardAccount>> {
     let mut reward_accounts = HashSet::default();
     let epoch_height = instance.epoch_height;
 
     let Some(epoch_height) = epoch_height else {
-        bail!("epoch height not set");
+        tracing::warn!("epoch height is not set. returning empty reward_header_dependencies");
```

Comment:
Nit: idk if this should be a warn log because it can happen in totally normal usage prior to rewards. Maybe info?